tests/nnfw_api/src/GenModelTests/one_op_trains/Conv2D.test.cc

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "GenModelTrain.h"
+
+TEST_F(GenModelTrain, OneOp_Conv2D)
+{
+  CirclePlusGen cgen;
+
+  int in = cgen.addTensor({{1, 5, 5, 1}, circle::TensorType::TensorType_FLOAT32});
```

Comment:
Is there a reason to use `int` here? 
Since `addTensor` return uint32_t, I thought that the type might be `uint32_t` either.

https://github.com/Samsung/ONE/blob/31c728d79bd8e3c91fb0b85a2cb7251c22fc9f82/tests/nnfw_api/lib/CircleGen.cc#L40C1-L41C1 

```suggestion
  uint32_t in = cgen.addTensor({{1, 5, 5, 1}, circle::TensorType::TensorType_FLOAT32});
```